Documentation/Examples/MSBuild/DeterministicBuild/Directory.Build.props

```diff
@@ -8,6 +8,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="all" />
+    <!-- https://devblogs.microsoft.com/dotnet/announcing-dotnet-8-preview-5/#sdk-source-link-is-part-of-the-net-sdk -->
```

Comment:
do we need an issue linked for this one?

Reply:
This hint can be removed as soon as coverlet uses net8.0 SDK. Microsoft released .NET 8.0.0-rc.1 this month.